ingestion/src/metadata/orm_profiler/orm/converter.py

```diff
@@ -146,8 +149,11 @@ def ometa_to_orm(table: Table, metadata: OpenMetadata) -> DeclarativeMeta:
         for idx, col in enumerate(table.columns)
     }
 
+    orm_database_name = get_orm_database(table, metadata)
     orm_schema_name = get_orm_schema(table, metadata)
-    orm_name = f"{orm_schema_name}_{table.name.__root__}".replace(".", "_")
+    orm_name = f"{orm_database_name}_{orm_schema_name}_{table.name.__root__}".replace(
```

Comment:
I am guessing that for sources such as MySQL, the orm_database_name will always be default right?